src/main/java/duke/Event.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/** Represents a task of type event.
+ */
+public class Event extends Task {
+    private LocalDate dateTime;
+
+    /** Creates a task of type event with the
+     * specified description and the date of the event.
+     * @param description is the description of the task.
+     * @param dateTime is the date that the event occurs.
+     */
+    public Event(String description, LocalDate dateTime) {
+        super(description);
+        this.dateTime = dateTime;
+    }
+
+    /** Produces string to be printed.
+     * @return a string that represents the task.
+     */
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " (at: " + dateTime.format(DateTimeFormatter.ofPattern("MMM d yyyy")) + ")";
```

Comment:
Great polymorphism and usage of super class methods. Perhaps you can handle when user inputs startTime-endTime as defined in the definition of an event. But overall great job.

Reply:
Oh yeah! A suggestion is to handle wrong date formats as it will throw exceptions, it may be handled somewhere else in your code, so if it is, then ignore this comment :)